lib/node_modules/@stdlib/math/base/special/acoversinf/README.md

```diff
@@ -18,7 +18,7 @@ limitations under the License.
 
 -->
 
-# Arccoversinef
+# acoversinf
 
 > Compute the [inverse coversed sine][inverse-coversed-sine] of a single-precision floating-point number (in radians).
 
```

Comment:
@gunjjoshi Would you mind fixing the alias in the math equations to `acoversin`? We don't need to use the `f` suffix in math equations.